src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp

```diff
@@ -481,6 +481,14 @@ namespace AppInstaller::Manifest
             );
     }
 
+    bool DoesInstallerTypeIgnoreScopeFromManifest(InstallerTypeEnum installerType)
+    {
+        return (
+            installerType == InstallerTypeEnum::Msix ||

```

Comment:
I don't think `MSIX` is ready to be there yet, I was just saying that I want to include it eventually (when the work to actually support machine install is done).